benchmarks/pool_stress_test.cpp

```diff
@@ -0,0 +1,93 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include <iostream>
+#include <chrono>
+#include <string>
+#include <random>
+#include <numeric>
+
+#include "umpire/config.hpp"
+#include "umpire/ResourceManager.hpp"
+#include "umpire/Allocator.hpp"
+#include "umpire/strategy/MixedPool.hpp"
+
+#define CONVERT 1000000 //convert sec (s) to microsec (us)
```

Comment:
This does not need to be global. This should be moved within `run_test`.

Reply:
Done